benchmark/scenarios/write_request_load.py

```diff
@@ -0,0 +1,289 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Read request load scenario for the control service benchmarks.
+"""
+from collections import deque
+from itertools import repeat
+
+from zope.interface import implementer
+import eliot
+
+from twisted.internet.defer import CancelledError, Deferred, succeed
+from twisted.internet.task import LoopingCall
+
+from flocker.common import loop_until, timeout
+
+from .._interfaces import IScenario
+
+DEFAULT_SAMPLE_SIZE = 5
+
+# XXX make it a common class for Read and Write scenarios.
+# Note that the docstrings are not up-to-date because they are being updated
+# in the Read scenario
+class WRateMeasurer(object):
+    """
+    Measures the rate of requests in requests per second
```

Comment:
Why does this class have `W` in its name?
That's not particularly clear from the name itself and from the doc-string.
I think that this is worth explaining.